### PR TITLE
Change for Census england print spec

### PR DESCRIPTION
**Pull request by LukeLoze1**
# Motivation and Context
For the new Census spec with require a new print file type

# What has changed
Added in a new template and mappings.
Copied the action-api xsd in locally
Changed pom to compile xsds
Removed unused endpoints
Removed unused dependencies from pom

# How to test?
This will require the updated actionsvc & acceptance tests.  (links below)
Should work locally with the acceptance tests

# Links
https://github.com/ONSdigital/census-rm-action-service/pull/3
https://github.com/ONSdigital/census-rm-acceptance-tests

https://trello.com/c/e9iWXgje/517-rmc-83-load-hh-sample-and-generate-print-file-to-census-spec-13

### CODE REVIEW

**Comment by AdamHawtin** [comment]
[RMC-83 Load HH sample and generate print file to Census spec [13]](https://trello.com/c/e9iWXgje/517-rmc-83-load-hh-sample-and-generate-print-file-to-census-spec-13)

File: src/main/resources/database/changes/release-16/pgcrypto_fix.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+DROP EXTENSION IF EXISTS pgcrypto CASCADE;
```
Comment (AdamHawtin) [inline src/main/resources/database/changes/release-16/pgcrypto_fix.sql:1]: Shouldn't need this anymore?

Comment (LukeLoze1) [reply src/main/resources/database/changes/release-16/pgcrypto_fix.sql:1]: I think we do, the next script tries to set it up.  Uses it altering the actionexporter.address table.

File: pom.xml [hunk 2]
```diff
@@ -308,7 +291,7 @@
                         </goals>
                         <phase>package</phase>
                         <configuration>
-                            <repository>${project.artifactId}</repository>
+                            <repository>eu.gcr.io/census-rm-ci/rm/${project.artifactId}</repository>
```
Comment (AdamHawtin) [inline pom.xml:294]: The step of the travis build which tries to retag the built image name can be deleted with this change.

File: src/test/java/uk/gov/ons/ctp/response/action/export/service/TemplateServiceIT.java [hunk 3]
```diff
@@ -266,12 +158,28 @@ private String getLatestSftpFileName() throws IOException {
 
     String sftpPath = "Documents/sftp/";
     ChannelSftp.LsEntry[] sftpList = defaultSftpSessionFactory.getSession().list(sftpPath);
+
+    System.out.println("Latest File Length list: " + sftpList.length);
```
Comment (liamtoozer) [inline src/test/java/uk/gov/ons/ctp/response/action/export/service/TemplateServiceIT.java:162]: Is it worth putting this into a log?

**Review by liamtoozer — APPROVED** [review]
LGTM!

**Review by NickGrantX — APPROVED** [review]
LGTM 👍

File: .travis.yml [hunk 4]
```diff
@@ -8,7 +8,7 @@ jdk: openjdk8
 
 env:
   global:
-    - DESTINATION_IMAGE_NAME="$DOCKER_GCP_LOCATION/census-rm-actionexportersvc"
+    - DESTINATION_IMAGE_NAME="eu.gcr.io/census-rm-ci/rm/census-rm-actionexportersvc"
```
Comment (liamtoozer) [inline .travis.yml:11]: ~~Should we remove this `env.global` section now as `DESTINATION_IMAGE_NAME` is only being used once? I guess it does give it a meaningful name, though.~~

@AdeHarrison picking these changes up 👍

**Review by hbrace — APPROVED** [review]

File: src/test/java/uk/gov/ons/ctp/response/action/export/service/TemplateServiceIT.java (hunk 3 above)
Comment (LukeLoze1) [reply src/test/java/uk/gov/ons/ctp/response/action/export/service/TemplateServiceIT.java:162]: Probably best to just remove, I'll do that

**Pull request merged**